core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -1169,6 +1169,103 @@ class SQLPlanParserSuite extends BasePlanParserSuite {
     }
   }
 
+  test("MonthsBetween is supported in ProjectExec") {
+    TrampolineUtil.withTempDir { parquetoutputLoc =>
+      TrampolineUtil.withTempDir { eventLogDir =>
+        val (eventLog, _) = ToolTestUtils.generateEventLog(eventLogDir,
+          "ProjectExprsSupported") { spark =>
```

Comment:
`ProjectExprsSupported` ->  if we end up having 3 unit tests, it will be nice to have different names for the app. Different names for unit-tests help when we debug and extract the eventlogs for further debugging and testing. the app name then will be used to link back to the unit-test that created that eventlog.